crates/formats/src/io_ext/seek.rs

```diff
@@ -0,0 +1,22 @@
+use std::io::{self, ErrorKind, Seek, SeekFrom};
+
+pub trait SeekExt: Seek {
+    fn seek_until_alignment(&mut self, alignment: usize) -> io::Result<usize>;
+}
+
+impl<T: Seek> SeekExt for T {
```

Comment:
I'd make this work over `Read` and `std::io::copy(&mut read.take(required), &mut std::io::sink())` so we can use this on readers that don't support seeking.

Reply:
[65a45f6](https://github.com/soulsmods/fstools-rs/pull/31/commits/65a45f6419e66c2e588cafebc48047ea7518d73c) removes it and replaces it with a call to read padding which I think is a bit better